### PR TITLE
only print themes hint in interactive mode

### LINKED ISSUE #1437: extra themes help text to stderr
Can the help text for `--list-themes` from #1353 be written to stderr?
I'm using the command to retrieve themes for [completion](https://github.com/rsteube/carapace-bin/blob/master/completers/bat_completer/cmd/root.go#L86) and that interfers a bit.

### PR DESCRIPTION
**Pull request by rsteube**
fixes #1437

### CODE REVIEW

File: src/bin/bat/main.rs [hunk 1]
```diff
@@ -200,19 +200,19 @@ pub fn list_themes(cfg: &Config) -> Result<()> {
                 .ok();
             writeln!(stdout)?;
         }
+        writeln!(
+            stdout,
+            "Further themes can be installed to '{}', \
+        and are added to the cache with `bat cache --build`. \
```
Comment (keith-hall) [inline src/bin/bat/main.rs:206]: I guess this should be indented an extra level, so everything is the same level like it was before

Comment (rsteube) [reply src/bin/bat/main.rs:206]: oh, thought i had executed `:Format` - will have a look at it

Comment (sharkdp) [reply src/bin/bat/main.rs:206]: I could imagine `cargo fmt` doesn't fix that because it's a multiline string

Comment (rsteube) [reply src/bin/bat/main.rs]: ah ok - fixed it

**Review by sharkdp — APPROVED** [review]

**Review by keith-hall — APPROVED** [review]
Thanks for your contribution!

**Comment by keith-hall** [comment]
Is it worth including this in the changelog? I was thinking yes, personally :)

**Comment by sharkdp** [comment]
Right, that would be great: https://github.com/sharkdp/bat/blob/master/CONTRIBUTING.md#add-an-entry-to-the-changelog

**Pull request merged**